### PR TITLE
Repo_Manage_Section_UI

### PR DESCRIPTION
**Pull request by aman1376**
## Changes Proposed

Created the UI for repository manage section

Screenshot (54)

## Check List

- [X] The title of my pull request is a short description of the changes
- [X] This PR relates to some issue: 
- [X] I have documented the changes made (if applicable)
- [ ] I have covered the changes with unit tests

### CODE REVIEW

**Comment by aman1376** [comment]
@chad11111 please have a look.

File: dashboard/src/Repo_Manage_Section.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    font-weight:bold;
+}
+.Card>p{
+    text-align: justify;
+    margin:5px 25px 0px 5px;
+    padding: 5px;
+}
+.chart{
+    color:#00ff00;
+}
```
Comment (chad11111) [inline dashboard/src/Repo_Manage_Section.css:65]:
```suggestion
}

```

File: dashboard/src/Repo_Manage_Section.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+
+import "./Repo_Manage_Section.css"
+function Repo_Manage_Section():JSX.Element {
```
Comment (chad11111) [inline dashboard/src/Repo_Manage_Section.tsx:3]: please rename this to `RepoManageSection`

File: dashboard/src/App.tsx [hunk 3]
```diff
@@ -1,48 +1,10 @@
...
-          <a
-            className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
-            href="http://localhost:6006"
-          >
-            Run Storybook
-          </a>
-        </div>
-      </div>
-    </div>
+    <Repo_Manage_Section/>
```
Comment (chad11111) [inline dashboard/src/App.tsx:7]: remove the changes here

**Review by chad11111 — CHANGES_REQUESTED** [review]

**Comment by chad11111** [comment]
@aman1376 Please don't snake case (Repo_Manage_Section) only camel case (RepoManageSection)

**Comment by chad11111** [comment]
@aman1376 also please fix conflicts

**Comment by chad11111** [comment]
@aman1376 why did u close it ?

**Comment by aman1376** [comment]
i have created a new pull request RepoManageSection

**Pull request closed without merging**